cristin-organization/src/test/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandlerTest.java

```diff
@@ -210,6 +212,44 @@ void shouldReturnHttp404NotFoundWhenNonExistingIdentifier() throws Exception {
     }
 
 
+    @Test
+    void shouldReturnOrganizationFlatHierarchy() throws IOException, ApiGatewayException {
+
+        HttpClient httpClient = mock(HttpClient.class);
+        when(httpClient.sendAsync(any(), any())).thenThrow(new RuntimeException("This should Not happen!"));
+        output = new ByteArrayOutputStream();
+        fetchCristinOrganizationHandler = new FetchCristinOrganizationHandler(cristinApiClient, new Environment());
+
+        CristinOrganizationApiClient mySpy = spy(cristinApiClient);
+        final URI level1 = getCristinUri("185.90.0.0", UNITS_PATH);
+        doReturn(getSubSubUnit("unit_18_90_0_0.json")).when(mySpy).getSubSubUnitDtoWithMultipleEfforts(level1);
+        final URI level2a = getCristinUri("185.53.0.0", UNITS_PATH);
+        doReturn(getSubSubUnit("unit_18_53_0_0.json")).when(mySpy).getSubSubUnitDtoWithMultipleEfforts(level2a);
+        final URI level2b = getCristinUri("185.50.0.0", UNITS_PATH);
+        doReturn(getSubSubUnit("unit_18_50_0_0.json")).when(mySpy).getSubSubUnitDtoWithMultipleEfforts(level2b);
+        final URI level3 = getCristinUri("185.53.18.0", UNITS_PATH);
+        doReturn(getSubSubUnit("unit_18_53_18_0.json")).when(mySpy).getSubSubUnitDtoWithMultipleEfforts(level3);
+        final URI level4 = getCristinUri("185.53.18.14", UNITS_PATH);
+        doReturn(getSubSubUnit("unit_18_53_18_14.json")).when(mySpy).getSubSubUnitDtoWithMultipleEfforts(level4);
+
+        fetchCristinOrganizationHandler = new FetchCristinOrganizationHandler(cristinApiClient, new Environment());
```

Comment:
You are doing this two different places in the same method

Reply:
It's duplicated to have the same base return different with regards to query parameters